ultralytics/cfg/__init__.py

```diff
@@ -31,27 +31,30 @@
 
 # Define valid tasks and modes
 MODES = {"train", "val", "predict", "export", "track", "benchmark"}
-TASKS = {"detect", "segment", "classify", "pose", "obb"}
+TASKS = {"detect", "segment", "classify", "pose", "obb", "human"}
 TASK2DATA = {
     "detect": "coco8.yaml",
     "segment": "coco8-seg.yaml",
     "classify": "imagenet10",
     "pose": "coco8-pose.yaml",
     "obb": "dota8.yaml",
+    "human": "human8.yaml",
 }
 TASK2MODEL = {
     "detect": "yolov8n.pt",
     "segment": "yolov8n-seg.pt",
     "classify": "yolov8n-cls.pt",
     "pose": "yolov8n-pose.pt",
     "obb": "yolov8n-obb.pt",
+    "human": "yolov8n-human.pt",
 }
 TASK2METRIC = {
     "detect": "metrics/mAP50-95(B)",
     "segment": "metrics/mAP50-95(M)",
     "classify": "metrics/accuracy_top1",
     "pose": "metrics/mAP50-95(P)",
     "obb": "metrics/mAP50-95(B)",
+    "human": "metrics/mAP50-95(B)",
```

Comment:
@Laughing-q what metric should be we put here for human models?

Reply:
@glenn-jocher There are six types of accuracy for weight/height/gender/age/ethnicity respectively and I feel like randomly choose one of them is not that decent. Let's stick with `metrics/mAP50-95(B)` for now(since it's also a detection model after all),

Reply:
We can also use Macro F1 score as an overall metric for the attributes. Decisions is up to you @Laughing-q 👌🏻